tests/Base.t.sol

```diff
@@ -542,9 +542,10 @@ abstract contract Base_Test is Assertions, Constants, DeployOptimized, Merkle, F
         return computeMerkleVCAAddress(
             merkleVCAConstructorParams({
                 campaignCreator: users.campaignCreator,
+                endTime: RANGED_STREAM_END_TIME,
                 expiration: EXPIRATION,
                 merkleRoot: MERKLE_ROOT,
-                schedule: merkleVCASchedule(),
+                startTime: RANGED_STREAM_START_TIME,
```

Comment:
we definitely need a new constant for this because the VCA != ranged airdrop

Reply:
Similar to `VCA_FULL_AMOUNT`, I will add `VCA_END_TIME` and `VCA_START_TIME`.